configs/training_callbacks/weight_visualization.yaml

```
@@ -0,0 +1,9 @@
+# # Visualize frequently at the beginning, then less often
+weight_visualization:
+  _target_: openretina.callbacks.weight_visualization_callback.ConditionalWeightVisualizationCallback
+  every_n_epochs: 1
+  start_epoch: 0
+  call_on_train_epoch_end: true
+  call_on_validation_epoch_end: false
+  save_plots: true
+  folder_path: "${paths.output_dir}/weight_plots/"
```

```suggestion
  folder_path: "${paths.output_dir}/weight_plots/"

```
new line at end of file